packages/core/src/index.ts

```diff
@@ -45,10 +45,12 @@ export {
 export {default as Head} from "next/head"
 
 export {default as Link} from "next/link"
+export {LinkProps} from "next/link"
```

Comment:
```suggestion
export {default as Link, LinkProps} from "next/link"
```

Reply:
Huh, I was going to do it like this but TS complained locally so I ended up not doing it.
It's much cleaner like this though, thanks for taking care of that. 👍